lib/codecs/xsdDateTime.js

```diff
@@ -0,0 +1,21 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+export class XsdDateTimeCodec {
+  constructor() {
+  }
+
+  set({value}) {
+    this.value = value;
+  }
+
+  encodeCBOR(gen) {
+    const encodedValue = new Date(this.value).valueOf() / 1000;
```

Comment:
Is it guaranteed that the value of `this.value` will not include fractional seconds?  If not...
```suggestion
    const encodedValue = Math.ceil(new Date(this.value).valueOf() / 1000);
```

Reply:
Yes, guaranteed that there won't be fractional seconds... valueOf() returns milliseconds since the epoch.

Reply:
Right, and the value returned from `.valueOf()` will always be evenly divisible by 1000?

Reply:
Ah, yes, it will be because ISO8601 doesn't support millis, IIRC. That said, this does feel dangerous, doesn't it.

Reply:
Fixed in dcc92ad5ea48fc45ad7a62d315935f75c91b305c.